docs/pages/access-controls/access-lists/reference.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Access List Reference
+description: An explanation and overview of Access Lists in Teleport.
+---
+
+<Admonition type="warning">
+Access Lists are currently in Preview mode.
+</Admonition>
+
+Access Lists allow Teleport users to be granted long term access to resources
+managed within Teleport. With Access Lists, administrators and access list
+owners can regularly audit and control membership to specific roles and
+traits, which then tie easily back into Teleport's existing RBAC system.
+
+## What do Access Lists do?
+
+Access Lists grant roles and traits to users on a long lived basis. Users who are added
+to Access Lists as members and meet membership requirements will be granted these roles
+and traits upon login in Teleport. By combining this functionality with an auditing
+functionality, Teleport administrators will have an audit trail for long lived access
+granted to users of Teleport.
+
+## How do Access Lists compare to Access Requests?
+
+Access Lists are intended for longer lived access to Teleport resources and Access Requests
+are intended for temporary elevation of privileges. Access conveyed by an Access List is
+expected to live on the order of months and access conveyed by an Access Request is intended
+to live on the order of hours or days.
+
+## Access List Ownership
+
+Access List owners are Teleport users who are granted special privileges over
+an Access List. These owners are defined explicitly as part of the Access List, and
+must be added by a Teleport administrator. Owners must meet requirements in order for
```

Comment:
Can we clarify who can add owners? There is not a formal concept of an "administrator" in Teleport.